tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist =
+    py{36,37,38}-test{,-all,-dev,-lts}
+    build_docs
+    style
```

Comment:
For other packages I decided to go with ``codestyle`` here rather than just ``style``, so maybe change for consistency? (I'll change it in the migration guide too)